python/tvm_ffi/module.py

```diff
@@ -133,8 +134,7 @@ def implements_function(self, name: str, query_imports: bool = False) -> bool:
 
         Returns
         -------
-        b
-            True if module (or one of its imports) has a definition for name.
+        True if module (or one of its imports) has a definition for name.
```

Comment:
there's no need to name an extra variable - the return type is derived from function signature